pages/common/black.md

```diff
@@ -0,0 +1,24 @@
+# black
+
+> A speedy Python auto code formatter.
+> More information: <https://github.com/psf/black>.
+
+- Auto-format a file or entire directory:
+
+`black {source_file_or_directory}`
+
+- Format the code passed in as a string:
+
+`black -c {{source_file_or_directory}}`
+
+- Output a diff for each file on stdout:
+
+`black --diff {{source_file_or_directory}}`
+
+- Don't write the files back, just return the status:
```

Comment:
I feel like we should always start or descriptions with a verb.
```suggestion
- Return the status without writing the files back:
```
Not sure if that's the best way to rephrase it, though.